scripts/pi-hole/php/customdns.php

```diff
@@ -6,134 +6,10 @@
 
     switch ($_REQUEST['action'])
     {
-        case 'get':     echo json_encode(echoCustomDNSEntries());    break;
-        case 'add':     echo json_encode(addCustomDNSEntry());      break;
-        case 'delete':  echo json_encode(deleteCustomDNSEntry());   break;
+        case 'get':        echo json_encode(echoCustomDNSEntries());      break;
+        case 'add':        echo json_encode(addCustomDNSEntry());         break;
+        case 'delete':     echo json_encode(deleteCustomDNSEntry());      break;
```

Comment:
Added whitespace is intentional?

Reply:
I added a `deleteall` in between, but found how to do it without.
I'll fix the extra spaces, thanks.